analysis/org.eclipse.tracecompass.analysis.profiling.core/src/org/eclipse/tracecompass/internal/analysis/profiling/core/callgraph2/AggregatedCalledFunction.java

```diff
@@ -293,4 +293,24 @@ public AggregatedCalledFunctionStatistics getFunctionStatistics() {
     public String toString() {
         return "Aggregate Function: " + getObject() + ", Duration: " + getDuration() + ", Self Time: " + fSelfTime + " on " + getNbCalls() + " calls"; //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$ //$NON-NLS-4$ //$NON-NLS-5$
     }
+    /**
```

Comment:
There should a space in between those two functions. The file should be reformatted using eclipse format tool (ctrl+shift+f)